aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -1695,73 +1659,6 @@ module aptos_framework::stake {
         leave_validator_set(&validator, validator_address);
     }
 
-    #[test(
-        aptos_framework = @0x1,
-        core_resources = @core_resources,
-        validator_1 = @0x1,
-        validator_2 = @0x2,
-        validator_3 = @0x3,
-        validator_4 = @0x4,
-        validator_5 = @0x5
-    )]
-    public entry fun test_staking_is_sorted_by_address(
```

Comment:
can you keep this test, with asserting joining order, not sorted?

i.e. if nothing else, we need to make sure as people leave, that validator_index stays consecutive